src/netif.c

```diff
@@ -5110,6 +5212,14 @@ static int netif_sockopt_set(sockoptid_t opt, const void *in, size_t inlen)
             ret = set_bond(port, in);
             break;
         }
+        case SOCKOPT_NETIF_SET_ZERO:
```

Comment:
Do not use an isolate `ZERO` SOCKOPT to reset stats statistics. 
You can reset lcore statistics in `set_lcore` by `dpip link show cpu -s -Z`,  reset port statistics in `set_port` by `dpip link show -s -Z`, reset inetaddr statistics by `dpip adds show -Z`.